index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <title>Portfolio</title>
+        <meta name="description" content="">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link rel="stylesheet" href="./style.css">
+        <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    </head>
+    <body>
+        <section class="appbar">
+            <div class = "toolbar">
+                <div class="mylogo">
+                    My Logo
+                </div>
+                <nav class="menu-container">
+                    <a href="" class="hamburger" ">
+                        <i class="fa fa-bars"></i>
+                    </a> 
+                </nav>
+            </div>
+        </section>
+        <section class="myInfo" id="home">
+            <div class="hero-text">
+                <h1 class="hero_title">I'm Reza.<br>Glad to see you!</h1>
+                <p class="hero_desc">
+                    I’m a software developer! I can help you build a product , feature or website. Look through 
+                    some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hesitate to contact me.
+                    
+                </p>
+            </div>
+            <div class="contact">
+                <div class="contact-text">Let's Connect</div>
+                <div class="social-icons">
+                    <span>
+                        <a href="" alt="twitter"><img src="./icons/twitter.svg" /></a>
+                    </span>
+                    <span>
+                        <a href="" alt="linkedin"><img src="./icons/Linkedin icon.svg" /></a>
+                    </span>
+                    <span>
+                        <a href="" alt="medium"><img src="./icons/medium.svg" /></a>
+                    </span>
+                    <span>
+                        <a href="" alt="github"><img src="./icons/github.svg" /></a>
+                    </span>
+                    <span>
+                        <a href="" alt="angellist"><img src="./icons/angellist.svg" /></a>
+                    </span>
+                </div>
```

Comment:
- [ ] For the social icons, kindly note that you have to use a list (`ul` and `li`) to lay your icons instead of using `div` and `span` this will be more professional, and more appropriate as a semantic structure, so, the list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag)